htlcswitch/link_test.go

```diff
@@ -975,6 +975,48 @@ func TestUpdateForwardingPolicy(t *testing.T) {
 	default:
 		t.Fatalf("expected FailFeeInsufficient instead got: %v", err)
 	}
+
+	// Reset the policy so we can then test updating the max HTLC policy.
+	n.secondBobChannelLink.UpdateForwardingPolicy(n.globalPolicy)
+
+	// As a sanity check, ensure the original payment now succeeds again.
+	_, err = makePayment(
+		n.aliceServer, n.carolServer, firstHop, hops, amountNoFee,
+		htlcAmt, htlcExpiry,
+	).Wait(30 * time.Second)
+	if err != nil {
+		t.Fatalf("unable to send payment: %v", err)
+	}
+
+	// Now we'll update Bob's policy to lower his max HTLC to an extent
+	// that'll cause him to reject the same HTLC that we just sent.
+	newPolicy = n.globalPolicy
+	newPolicy.MaxHTLC = amountNoFee - 1
+	n.secondBobChannelLink.UpdateForwardingPolicy(newPolicy)
```

Comment:
check err?

Reply:
Sorry, `UpdateForwardingPolicy` doesn't return an error, am I missing something?

Reply:
Hmm, don't know where that comment came from.